proxy/src/executor.rs

```diff
@@ -0,0 +1,252 @@
+#[cfg(test)]
+mod tests;
+
+use std::mem;
+use std::sync::Arc;
+
+use alloy_consensus::TxEnvelope;
+use alloy_rlp::Encodable;
+use anyhow::Context;
+
+use common::ethnum::U256;
+use common::evm_loader::config::ACCOUNT_SEED_VERSION;
+use common::neon_instruction::tag;
+use common::neon_lib::commands::emulate::EmulateResponse;
+use common::neon_lib::types::{Address, TxParams};
+use common::solana_sdk::instruction::{AccountMeta, Instruction};
+use common::solana_sdk::pubkey::Pubkey;
+use common::solana_sdk::signature::Keypair;
+use common::solana_sdk::signature::Signature;
+use common::solana_sdk::signer::Signer;
+use common::solana_sdk::system_program;
+use common::solana_sdk::transaction::Transaction;
+use solana_api::solana_api::SolanaApi;
+
+use crate::convert::ToNeon;
+use crate::neon_api::NeonApi;
+
+#[derive(Clone)]
+pub struct Executor {
+    program_id: Pubkey,
+
+    neon_api: NeonApi,
+    solana_api: SolanaApi,
+
+    treasury_pool_count: u32,
+    treasury_pool_seed: Vec<u8>,
+
+    operator: Arc<Keypair>,
+    operator_addr: Address,
+}
+
+impl Executor {
```

Comment:
Taking into account TODOs and FIXMEs, this looks good enough to me 👍 

However, I don’t quite understand the scope of this PR (the corresponding issue is empty). Are we supposed to implement sending a transaction through a (currently non-existent) mempool as it is done in Python, or is that out of the scope of this PR? Or maybe it’s entirely out of the scope of the project?